### PR TITLE
Draft updated README with usage instructions

### PR DESCRIPTION
**Pull request by BrooklinJazz**
README format based on https://github.com/liveview-native/liveview-native-swiftui-mapkit

## Desired Feedback
* Any missing instructions or usage steps?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,74 @@
 # LiveViewNativeLiveForm
 
-Supporting library for [LiveViewNative SwiftUI](https://github.com/liveviewnative/liveview-client-swiftui) that provides forms that group multiple pieces of data together and allow them to be submitted in a single event.
+## About
+
+`liveview-native-forms` is an add-on library for [LiveView Native SwiftUI](https://github.com/liveviewnative/liveview-client-swiftui). It adds support for conveniently working with control views grouped within a single form so you can use Phoenix [Form Bindings](https://hexdocs.pm/phoenix_live_view/form-bindings.html) within a LiveView Native application.
```
Comment (carson-katri) [inline README.md:5]:
```suggestion
`liveview-native-live-form` is an add-on library for [LiveView Native SwiftUI](https://github.com/liveviewnative/liveview-client-swiftui). It adds support for conveniently working with control views grouped within a single form so you can use Phoenix [Form Bindings](https://hexdocs.pm/phoenix_live_view/form-bindings.html) within a LiveView Native application.
```

File: README.md [hunk 2]
```diff
@@ -1,20 +1,74 @@
...
-        MyRegistry,
-        LiveFormRegistry<Self>
-    >
-}
-```
-And then use that registry when creating the `LiveSessionCoordinator`:
 ```swift
+//
+//  ContentView.swift
+//  KinoLiveView
```
Comment (carson-katri) [inline README.md:23]: Might be good to remove this header comment.

File: README.md [hunk 3]
```diff
@@ -1,20 +1,74 @@
...
+```elixir
+defmodule AppWeb.FormLive do
+  use Phoenix.LiveView
+  use LiveViewNative.LiveView
+
+  @impl true
+  def render(%{format: :swiftui} = assigns) do
+    ~SWIFTUI"""
+    <LiveForm id="my-form" phx-submit="submit">
+      <TextField name="my-text" >Placeholder</TextField>
```
Comment (carson-katri) [inline README.md:53]:
```suggestion
      <TextField name="my-text">Placeholder</TextField>
```

**Review by carson-katri — CHANGES_REQUESTED** [review]

**Comment by BrooklinJazz** [comment]
@carson-katri all feedback has been addressed.

**Comment by bcardarella** [comment]
Closing in favor of using the newer syntax

**Pull request closed without merging**